src/locales/en.json

```diff
@@ -1,16 +1,17 @@
 {
 	"Dev": "Developer",
-	"Club": "Student Club",
+	"Club": "Groups",
```

Comment:
make the arabic version consistent to this or the other way around

you can't be a student club in arabic but not in english